beerus_cli/src/model.rs

```diff
@@ -120,6 +127,11 @@ impl Display for CommandResponse {
             CommandResponse::EthereumQueryBlockNumber(block_number) => {
                 write!(f, "{block_number}")
             }
+            // Print the code of a contract in 256bits vector
+            // Result looks like: [1,1,10,ff]
+            CommandResponse::EthereumQueryCode(code) => {
+                write!(f, "{code:?}")
```

Comment:
add opt to save the file (ex: `-o code.json`) or add issue and `TODO`

Reply:
The idea would be that we will have a bool or enum option to return a json file aswell? Would this option be taken as argument or just manually change?

Reply:
No it'd be an optional path if provided then the code is dumped into the file if not provided simply print it as you're doing now.